.github/workflows/build.yml

```diff
@@ -80,3 +80,5 @@ jobs:
         run: test/app/test.sh
         shell: bash
         if: "env.GIT_DIFF != ''"
+      - name: "Go vulnerability check"
+        run: make vulncheck
```

Comment:
Is there any particular reason why the vulnerability check is part of the build workflow?

I'd prefer it if it were its own standalone workflow, so we can clearly distinguish between build failures and vulnerability-related failures in CI.

Reply:
You def don't want builds to succeed with vulnerabilities. However if you want the override, just caution that many passes can be made optional and if builds succeed, people can ignore them. Help me with a confirmation that you'd prefer it as a different pass.

Reply:
We'd just add the vulnerability check workflow as a required workflow, like our other required workflows.

Not sure why the Build workflow isn't required at the moment :smile: I'll fix that.

Reply:
> Not sure why the Build workflow isn't required at the moment

Ah, because the v0.34 branch doesn't have a separate "Build" workflow (it's part of the "Tests" workflow there).

Another reason to add this vulnerability check as a distinct workflow, because then it can be automatically backported with minimal effort to the release branches.